.changeset/tame-geese-change.md

```diff
@@ -0,0 +1,12 @@
+---
+'@shopify/hydrogen-react': major
+'@shopify/hydrogen': major
+---
```

Comment:
Doing this will cause our package versions to be something like `2024.x.x` I believe. 

What I've had to do in the past is manually update everything, for example `2023.4.0`, and then once it's all ready, push a changeset with a `patch` change.